src/dune_rules/alias0.ml

```diff
@@ -23,6 +23,7 @@ let doc_json = standard "doc-json"
 let doc_new = standard "doc-new"
 let check = standard "check"
 let install = standard "install"
+let pkg_install = Alias.Name.of_string "pkg-install"
```

Comment:
Something of a bikeshedding comment but: Are we going with `@pkg-install`? I think `@install-pkg` fits better with `@install`.

Reply:
I don't have any opinion on this. My main point is that I don't want the user to confuse the behaviour of `@install` with `@install-pkg`.

Reply:
I think I remember @rgrinberg proposing having "install" in the name because it is conceptually similar to the `@install` target but I am hazy on the exact wording.

Reply:
Let's keep the `pkg` prefix then.